samples/peripheral/radio_test/CMakeLists.txt

```
@@ -6,11 +6,11 @@
 
 cmake_minimum_required(VERSION 3.20.0)
 
-if (DEFINED SHIELD AND ${BOARD} STREQUAL "nrf5340dk_nrf5340_cpunet")
+if(DEFINED SHIELD AND "${BOARD}" STREQUAL "nrf5340dk_nrf5340_cpunet")
   string(REPLACE " " ";" SHIELD_LIST "${SHIELD}")
   foreach(sh IN_LIST SHIELD_LIST)
```

There is a typo here, this worked previously by coincident. Now with quotes it does not work because of this bug.
Could you fix it also?
```suggestion
  foreach(sh IN LISTS SHIELD_LIST)
```

---

Fixed